src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
+  shape: 'string';
+  color: 'string';
```

Comment:
Is cannot be just any string 
![image](https://user-images.githubusercontent.com/97547824/236891834-2ad5abf6-e360-433d-9152-968284d60119.png)